tool/cli.py

```diff
@@ -129,6 +129,34 @@ def list_labels(service, repo):
     ], tablefmt="fancy_grid"))
 
 
+@click.command(name="list-tags",
+               help="List tags for the project. "
+                    "This is how you can select a repository for Github: <owner>/<project>.")
+@click.option('--service', "-s", type=click.STRING, default="github",
+              help="Name of the git service (e.g. github/gitlab).")
+@click.argument('repo', type=click.STRING, required=False)
+def list_tags(service, repo):
+    """
+    List the tags for the selected repository, default to repo in $PWD
+    """
+    app = App()
+    if repo:
+        serv = app.get_service(service, repo=repo)
+    else:
+        serv = app.guess_service()
+    repo_tags = serv.list_tags()
+    if not repo_tags:
+        print("No tags.")
+        return
+    print(tabulate([
+        (
+            tag.name,
+            tag.tarball_url
```

Comment:
I am not sure about the `tarball_url`, is it usefull? What about using the web url (e.g. `https://github.com/user-cont/colin/releases/tag/0.3.0`)?  (I am not sure, if there is an API for this, but we can probably guess it.)

(Or add an option to show tarball url instead of the web url)?

Reply:
Good point. Web url is better.